.env.example

```
@@ -1,16 +1,36 @@
-EXPO_PUBLIC_FIREBASE_MEASUREMENT_ID=G-ABCDEFGHIJ
-EXPO_PUBLIC_FIREBASE_VAPID_KEY=your_vapid_key_for_web_push
-
-# API Configuration (if still using legacy backend)
-EXPO_PUBLIC_API_BASE_URL=https://api.elmec.com/api/v1
+# =============================================================================
+# PRODUCTION SUPABASE CONFIGURATION FOR ELMEC V2
+# =============================================================================
+
+# Public Configuration (Safe for client-side)
+NEXT_PUBLIC_SUPABASE_URL=https://aoghysichevnuaddsocl.supabase.co
+NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImFvZ2h5c2ljaGV2bnVhZGRzb2NsIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTcyMjQ5NjEsImV4cCI6MjA3MjgwMDk2MX0.SJE9eh1uQSXmYEXe2S6d9QZYta8V6AGkP1huj72DgrI
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_ANON_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)